apps/home/templates/apps_home/blocks/newsletter_block.html

```diff
@@ -0,0 +1,124 @@
+                            <label class="form-check-label" for="OPT_IN">{{ self.dsgvo_checkbox_label }}</label>
+                            <div id="OPT_IN-error" style="display: none" class="message message--error">{% trans 'Please accept the terms of conditions' %}</div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+                <div class="message--success" id="success-message" role="alert" style="display: none">
+                    <h3 class="title">{% trans 'Your subscription was successful.' %}</h3>
+                    <p>{% trans 'Your subscription to our E-Mail Newsletter was successful.' %}</p>
+                </div>
+                <div style="padding: 8px 0;">
+                  <div class="sib-form-block" style="text-align: right">
+                    <button class="sib-form-block__button sib-form-block__button-with-loader button" form="sib-form" type="submit">
+                      {% trans 'Subscribe' %}
+                    </button>
+                  </div>
+                </div>
+              </form>
+            </div>
+          </div>
+        </div>
+    </div>
+</section>
+
+<script>
+    function getInvalidFields() {
+        var email = document.getElementById('EMAIL');
+        var opt_in = document.getElementById('OPT_IN');
+
+        if (email.value.length > 0 && opt_in.checked) {
+            return false;
+        } else {
+            var invalids = []
+            email.value.length <= 0 && invalids.push(email);
+            !opt_in.checked && invalids.push(opt_in);
+            return invalids;
+        }
+    }
+
+    function setInvalidFields(fields, display) {
+        fields.forEach(function(field) {
+            if (display) {
+                field.classList.add('is-invalid');
+                var error = document.getElementById(field.id + '-error');
+                error.style.display = 'block';
+            } else {
+                field.classList.remove('is-invalid');
+                var error = document.getElementById(field.id + '-error');
+                error.style.display = 'none';
+            }
+        });
+    }
+
+    function submitForm(event) {
+        var prevInvalidFields = document.querySelectorAll('.is-invalid');
+        prevInvalidFields && setInvalidFields(prevInvalidFields, false);
+        var invalidFields = getInvalidFields();
+
+        if (invalidFields) {
+            setInvalidFields(invalidFields, true);
+        } else {
+            var payload = {
+                email: document.getElementById('EMAIL').value,
+                includeListIds: [parseInt('{{ self.sendinblue_list_id }}')],
+                templateId: parseInt('{{ self.sendinblue_template_id }}'),
+                redirectionUrl: '{{ self.sendinblue_email_link }}'
+            }
+
+            var http = new XMLHttpRequest();
+
+            var url = '{{ self.api_url }}';
```

Comment:
I thought to better put it into wagtail instead of into the pillar, so that when we hand this over, it is settable by editors. But actually when something changes, the code probably has to be touched anyway and we can remove it from the block.